README.md

```diff
@@ -92,9 +92,31 @@ docker-compose --help
    cd MLOps
    ```
    
-The server can be configured by modifying the environment file found at ```/mlflow_server/.env```. The environment variable provided are given as an example, and should not be used for a production deployment.
+2. The server should be configured by modifying the environment file found at ```/mlflow_server/.env```. The environment variable shown are given as an example, and should not be used for a production deployment.
 
-2. Navigate to the cloned code repository and start the server. Any docker images that are not present on your local system will be pulled from dockerhub (which might take a while).
+They are compulsory to be set otherwise the server will crash.
```

Comment:
Please replace with:
"Setting these variables is a requirement, the server will fail to start if they are undefined."